DOCKER/README.md

```diff
@@ -44,18 +44,12 @@ docker run -it --rm -v "/tmp:/tendermint" tendermint/tendermint init
 docker run -it --rm -v "/tmp:/tendermint" tendermint/tendermint node --proxy_app=kvstore
 ```
 
-## mintnet-kubernetes
+# Local cluster
 
-If you want to see many containers talking to each other, consider using [mintnet-kubernetes](https://github.com/tendermint/tools/tree/master/mintnet-kubernetes), which is a tool for running Tendermint-based applications on a Kubernetes cluster.
+To run a 4-node network, see the `Makefile` in the root of this repo and run:
 
-# License
```

Comment:
Why delete this?

Reply:
Because finding the LICENSE is a no-brainer, seems superfluous to have a link to it from the README inside a directory inside a repo. I doubt anyone has ever followed that link

![screenshot from 2018-05-23 07-46-36](https://user-images.githubusercontent.com/8304391/40422472-73108a38-5e5d-11e8-8cd2-44ebac04da1b.png)

Reply:
the readme is here https://hub.docker.com/r/tendermint/tendermint/